<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Home</title>

<script src="site_libs/header-attrs-2.8/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">reproduce_lucas_wflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/cancer-genomics/reproduce_lucas_wflow">
    <span class="fas fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-08-19
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 1 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>reproduce_lucas_wflow/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong4f74274"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 4f74274 </a>
</p>
</div>
<div id="strongRepositoryversionstrong4f74274" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 4f74274. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    analysis/.DS_Store
    Ignored:    code/.DS_Store
    Ignored:    data/.DS_Store

Untracked files:
    Untracked:  LICENSE
    Untracked:  README.Rmd
    Untracked:  analysis/SessionInfo.Rmd
    Untracked:  analysis/extended_fig10a.Rmd
    Untracked:  analysis/extended_fig10ab.Rmd
    Untracked:  analysis/extended_fig10cd.Rmd
    Untracked:  analysis/extended_fig11.Rmd
    Untracked:  analysis/extended_fig12.Rmd
    Untracked:  analysis/extended_fig2ab.Rmd
    Untracked:  analysis/extended_fig2c.Rmd
    Untracked:  analysis/extended_fig3.Rmd
    Untracked:  analysis/extended_fig4.Rmd
    Untracked:  analysis/extended_fig5.Rmd
    Untracked:  analysis/extended_fig6a.Rmd
    Untracked:  analysis/extended_fig6b.Rmd
    Untracked:  analysis/extended_fig6c.Rmd
    Untracked:  analysis/extended_fig7.Rmd
    Untracked:  analysis/extended_fig8.Rmd
    Untracked:  analysis/extended_fig9.Rmd
    Untracked:  analysis/fig2a.Rmd
    Untracked:  analysis/fig2b.Rmd
    Untracked:  analysis/fig2c.Rmd
    Untracked:  analysis/fig2c_pt2.Rmd
    Untracked:  analysis/fig3.Rmd
    Untracked:  analysis/fig4ab.Rmd
    Untracked:  analysis/fig4c.Rmd
    Untracked:  analysis/fig5a.Rmd
    Untracked:  analysis/fig5b.Rmd
    Untracked:  analysis/fig5cdef.Rmd
    Untracked:  analysis/fig6b.Rmd
    Untracked:  analysis/fig6cdefg.Rmd
    Untracked:  code/PlasmaTools.lucas/
    Untracked:  code/model_code/
    Untracked:  code/models_c1/
    Untracked:  code/models_c2/
    Untracked:  code/models_c3/
    Untracked:  code/plot-roc.r
    Untracked:  code/plot-roc_with_e9.r
    Untracked:  code/prepare_tcga_plot_data.r
    Untracked:  code/preprocessing/
    Untracked:  code/rlucas/
    Untracked:  code/rpcr/
    Untracked:  data/Extended Data Fig.12_recurrence_analysis copy.xlsx
    Untracked:  data/Extended Data Fig.12_recurrence_analysis.xlsx
    Untracked:  data/LUCAS_metadata.xlsx
    Untracked:  data/LUCAS_pts_50_80_py20_excluding prior cancers.xlsx
    Untracked:  data/TCGA_Lung/
    Untracked:  data/TNM_stage_DELFI_4a.xlsx
    Untracked:  data/TNM_stage_DELFI_4b.xlsx
    Untracked:  data/all_sclc_vs_all_healthy_fragment_size.txt
    Untracked:  data/all_sclc_vs_all_healthy_tfbs_coverage.txt
    Untracked:  data/ascl1_rel_cov_10v10.txt
    Untracked:  data/ascl1_tpm.txt
    Untracked:  data/cristiano-GCLOESS-lucas-cov-z-mito-orig-delfi473.rds
    Untracked:  data/cristiano-filtered-lucas2-cov-z-mito-gcloess-orig-delfi473.rds
    Untracked:  data/cristiano2019_tableS7.csv
    Untracked:  data/delfi_sensitivities.csv
    Untracked:  data/fig5_data.xlsx
    Untracked:  data/gc_correction.R/
    Untracked:  data/ichor, fragm length DELFI.xlsx
    Untracked:  data/id_subtype_key.txt
    Untracked:  data/lucas-holdout-preds.csv
    Untracked:  data/lucas_5mbs_delfi473.csv
    Untracked:  data/median_lengths.R/
    Untracked:  data/model_seq_glm.rds
    Untracked:  data/pcr_experiment.rda
    Untracked:  data/sclc_vs_other_fragment_size.txt
    Untracked:  data/score.seq_fig3b_filtered.rds
    Untracked:  data/sensitivity, specificity numbers in LUCAS_03_07_21.xlsx
    Untracked:  data/stats.rds
    Untracked:  data/ten_sample_rel_size.txt
    Untracked:  data/testing-set.csv
    Untracked:  data/tfbs_coverage.txt
    Untracked:  data/tpm.txt
    Untracked:  data/training-set.csv
    Untracked:  data/validation_preds.csv

Unstaged changes:
    Modified:   analysis/_site.yml
    Modified:   analysis/index.Rmd
    Modified:   analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
4f74274
</td>
<td>
shay-279
</td>
<td>
2021-08-19
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="detection-and-characterization-of-lung-cancer-using-cell-free-dna-fragmentomes" class="section level2">
<h2>Detection and characterization of lung cancer using cell-free DNA fragmentomes</h2>
<p>Abstract:</p>
Lung cancer remains the leading cause of cancer death world-wide, largely due to its late diagnosis. Non-invasive approaches for assessment of cell-free DNA (cfDNA) provide an opportunity for early detection and intervention that may have broader accessibility than current imaging approaches. Using a machine learning model for detecting tumor-derived cfDNA through genome-wide analyses of cfDNA fragmentation, we examined a prospective study of 365 individuals at risk for lung cancer (Lung Cancer Diagnostic Study, LUCAS), including 129 individuals ultimately diagnosed with lung cancer and 236 individuals determined to not have lung cancer. We externally validated the cancer detection model using an independent cohort of 385 non-cancer individuals and 46 predominantly early stage lung cancer patients. Combining fragmentation features with clinical risk factors and CEA levels followed by CT imaging detected 94% of patients with cancer across stages and subtypes, including 91% of stage I/II and 96% of stage III/IV, at 80% specificity. Genome-wide fragmentation profiles across ~13,000 ASCL1 transcription factor binding sites could be used to distinguish individuals with small cell lung cancer (SCLC) from those with non-small cell lung cancer (NSCLC) with high accuracy (AUC=0.98). Among individuals with lung cancer, a higher cfDNA fragmentation score was associated with tumor size and invasion, and represented an independent prognostic indicator of survival. These studies provide a facile approach for non-invasive detection of lung cancer and clinical management of this disease.
</center>
</div>
<div id="dependencies" class="section level1">
<h1>Dependencies</h1>
<p>A list of R-related dependencies for reproducing the figures in the LUCAS manuscript is available <a href="SessionInfo.html">here</a>. Most of the R packages are available from Bioconductor or CRAN. PlasmaTools.lucas and rlucas are available in the code/ folder.</p>
</div>
<div id="alignment-and-pre-processing" class="section level1">
<h1>Alignment and pre-processing</h1>
<div id="alignment" class="section level2">
<h2>Alignment</h2>
<p>The alignment of the sequenced fragments and pre-processing of the aligned reads are outside the scope of this workflowr repository. For processing unaligned reads, we used fastp (v0.20.0), bowtie2 (v 2.3.5.1), samtools (v 1.9), and sambamba (v 0.7.1). The following scripts that can be adapted for re-analysis of the LUCAS dataset:</p>
<ul>
<li><a href="../code/preprocessing/align.sh">align.sh</a>: aligns sequenced fragments to the Hg19 reference genome using bowtie2. Duplicates are flagged and removed with sambamba.</li>
</ul>
</div>
<div id="preprocessing" class="section level2">
<h2>Preprocessing</h2>
<p>See the <a href="../code/preprocessing/README.Rmd">code/preprocessing/README</a> for details.</p>
</div>
</div>
<div id="figures" class="section level1">
<h1>Figures</h1>
<ul>
<li><p>Figure 1: [<a href="figure/fig1.png">png</a> ]</p></li>
<li><p>Figure 2:</p>
<ul>
<li><p>2a [<a href="../analysis/fig2a.Rmd">R</a>, <a href="fig2a.html">html</a> , <a href="figure/fig2a.Rmd/fig2a-1.pdf">pdf</a> ]</p></li>
<li><p>2b [<a href="../analysis/fig2b.Rmd">R</a>,<a href="fig2b.html">html</a> , <a href="figure/fig2b.Rmd/heatmap-1.pdf">pdf</a> ]</p></li>
<li><p>2c (in two panels) [<a href="../analysis/fig2c.Rmd">R</a>,<a href="fig2c.html">html</a> , <a href="figure/fig2c.Rmd/heatmap3-1.pdf">pdf</a>, <a href="../analysis/fig2c_pt2.Rmd">R</a>, <a href="fig2c_pt2.html">html</a> , <a href="figure/fig2c_pt2.Rmd/plot-1.pdf">pdf</a> ]</p></li>
</ul></li>
<li><p>Figure 3:[<a href="../analysis/fig3.Rmd">R</a>, <a href="fig3.html">html</a>]</p>
<ul>
<li><p>3a [<a href="figure/fig3.Rmd/fig3A-1.pdf">pdf</a> ]</p></li>
<li><p>3b (in two parts) [<a href="figure/fig3.Rmd/roc_categories-1.pdf">pdf</a>, <a href="figure/fig3.Rmd/overall-1.pdf">pdf</a>]</p></li>
<li><p>3c [<a href="figure/fig3.Rmd/fig3C-1.pdf">pdf</a> ]</p></li>
</ul></li>
<li><p>Figure 4:</p>
<ul>
<li><p>4a, 4b (in two parts) [<a href="../analysis/fig4ab.Rmd">R</a>,<a href="fig4ab.html">html</a>, <a href="figure/fig4ab.Rmd/fig4a-1.pdf">pdf</a>, <a href="figure/fig4ab.Rmd/fig4b-1.pdf">pdf</a>]</p></li>
<li><p>4c [<a href="../analysis/fig4c.Rmd">R</a>, <a href="fig4c.html">html</a> , <a href="figure/fig4c.Rmd/survcurve_seq-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Figure 5:</p>
<ul>
<li><p>5a [<a href="../analysis/fig5a.Rmd">R</a>, <a href="fig5a.html">html</a> , <a href="figure/fig5a.Rmd/ascl1_expression-1.pdf">pdf</a>]</p></li>
<li><p>5b [<a href="../analysis/fig5b.Rmd">R</a>, <a href="fig5b.html">html</a> , <a href="figure/fig5b.Rmd/fig2-1.pdf">pdf</a>]</p></li>
<li><p>5c, 5d, 5e, 5f (in one panel) [<a href="../analysis/fig5cdef.Rmd">R</a>, <a href="fig5cdef.html">html</a> , <a href="figure/fig5cdef.Rmd/daniel_composite-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Figure 6:</p>
<ul>
<li><p>6a [<a href="figure/fig6a.png">png</a> ]</p></li>
<li><p>6b [<a href="../analysis/fig6b.Rmd">R</a>, <a href="fig6b.html">html</a> , <a href="figure/fig6b.Rmd/sens_table-1.pdf">pdf</a>]</p></li>
<li><p>6c, 6d, 6e, 6f, 6g (in one panel) [<a href="../analysis/fig6cdefg.Rmd">R</a>, <a href="fig6cdefg.html">html</a> , <a href="figure/fig6cdefg.Rmd/composite-1.pdf">pdf</a>]</p></li>
</ul></li>
</ul>
<p>Extended Figures:</p>
<ul>
<li><p>Extended Figure 1: [<a href="figure/extended_fig1.png">png</a>]</p></li>
<li><p>Extended Figure 2:</p>
<ul>
<li><p>2a, 2b (in one panel) [<a href="../analysis/extended_fig2ab.Rmd">R</a>, <a href="extended_fig2ab.html">html</a> , <a href="figure/extended_fig2ab.Rmd/pcafig-1.pdf">pdf</a>]</p></li>
<li><p>2c [<a href="../analysis/extended_fig2c.Rmd">R</a>, <a href="extended_fig2c.html">html</a> , <a href="figure/extended_fig2c.Rmd/profiles-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Extended Figure 3: [<a href="../analysis/extended_fig3.Rmd">R</a>, <a href="extended_fig3.html">html</a> , <a href="figure/extended_fig3.Rmd/heatmap-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 4: [<a href="../analysis/extended_fig4.Rmd">R</a>, <a href="extended_fig4.html">html</a>]</p>
<ul>
<li><p>4a [<a href="figure/extended_fig4.Rmd/smoking_seqmodel-1.pdf">pdf</a>]</p></li>
<li><p>4b, 4c (in one panel) [<a href="figure/extended_fig4.Rmd/gridgraphic-1.pdf">pdf</a>]</p></li>
<li><p>4d [<a href="figure/extended_fig4.Rmd/age_seqmodel-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Extended Figure 5: [<a href="../analysis/extended_fig5.Rmd">R</a>, <a href="extended_fig5.html">html</a> , <a href="figure/extended_fig5.Rmd/protein_scatter-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 6:</p>
<ul>
<li><p>6a [<a href="../analysis/extended_fig6a.Rmd">R</a>, <a href="extended_fig6a.html">html</a> , <a href="figure/extended_fig6a.Rmd/unnamed-chunk-1-1.pdf">pdf</a>]</p></li>
<li><p>6b [<a href="../analysis/extended_fig6b.Rmd">R</a>, <a href="extended_fig6b.html">html</a> , <a href="figure/extended_fig6b.Rmd/packages-4.pdf">pdf</a>]</p></li>
<li><p>6c [<a href="../analysis/extended_fig6c.Rmd">R</a>, <a href="extended_fig6c.html">html</a> , <a href="figure/extended_fig6c.Rmd/medians-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Extended Figure 7: [<a href="../analysis/extended_fig7.Rmd">R</a>, <a href="extended_fig7.html">html</a> , <a href="figure/extended_fig7.Rmd/extended_fig5-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 8: [<a href="../analysis/extended_fig8.Rmd">R</a>, <a href="extended_fig8.html">html</a> , <a href="figure/extended_fig8.Rmd/cea_nlratio-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 9 [<a href="../analysis/extended_fig9.Rmd">R</a>, <a href="extended_fig9.html">html</a> , <a href="figure/extended_fig9.Rmd/extfig9-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 10:</p>
<ul>
<li><p>10a [<a href="../analysis/extended_fig10a.Rmd">R</a>, <a href="extended_fig10a.html">html</a>,<a href="figure/extended_fig10a.Rmd/tfbs_fragment_size_10_sclc_vs_10_no_cancer-1.pdf">pdf</a>]</p></li>
<li><p>10b [<a href="../analysis/extended_fig10b.Rmd">R</a>, <a href="extended_fig10ab.html">html</a>,<a href="figure/extended_fig10ab.Rmd/sclc_no_cancer_no_benign-1.pdf">pdf</a>]</p></li>
<li><p>10c, 10d (in two parts) [<a href="../analysis/extended_fig10cd.Rmd">R</a>, <a href="extended_fig10cd.html">html</a>, <a href="figure/extended_fig10cd.Rmd/tfbs_fragment_size_sclc_vs_other-1.pdf">pdf</a>, <a href="figure/extended_fig10cd.Rmd/roc_fragment_size_sclc_vs_other-1.pdf">pdf</a>]</p></li>
</ul></li>
<li><p>Extended Figure 11 [<a href="../analysis/extended_fig11.Rmd">R</a>, <a href="extended_fig11.html">html</a> , <a href="figure/extended_fig11.Rmd/composite-1.pdf">pdf</a>]</p></li>
<li><p>Extended Figure 12 [<a href="../analysis/extended_fig12.Rmd">R</a>, <a href="extended_fig12.html">html</a>]</p>
<ul>
<li><p>12a [<a href="figure/extended_fig12.Rmd/fig12a-1.pdf">pdf</a>]</p></li>
<li><p>12b [<a href="figure/extended_fig12.Rmd/survcurve_seq-1.pdf">pdf</a>]</p></li>
</ul></li>
</ul>
<p>[<a href="SessionInfo.html">SessionInfo</a>] (packages and versions needed to implement the analysis)</p>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
