<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>SessionInfo</title>

<script src="site_libs/header-attrs-2.25/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">reproduce_lucas_wflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/cancer-genomics/reproduce_lucas_wflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">SessionInfo</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>reproduce_lucas_wflow/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongobjectspresent">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>Environment:</strong> objects present
</a>
</p>
</div>
<div id="strongEnvironmentstrongobjectspresent"
class="panel-collapse collapse">
<div class="panel-body">
<p>
The global environment had objects present when the code in the R
Markdown file was run. These objects can affect the analysis in your R
Markdown file in unknown ways. For reproduciblity it’s best to always
run the code in an empty environment. Use <code>wflow_publish</code> or
<code>wflow_build</code> to ensure that the code is always run in an
empty environment.
</p>
<p>
The following objects were defined in the global environment when these
results were created:
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Size
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
rv
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
2.9 Kb
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210819code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20210819)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210819code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210819)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomcancergenomicsreproducelucaswflowtreed71c8acbbc76834c3ab0e5c686ed4b664b054ddftargetblankd71c8aca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/cancer-genomics/reproduce_lucas_wflow/tree/d71c8acbbc76834c3ab0e5c686ed4b664b054ddf" target="_blank">d71c8ac</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomcancergenomicsreproducelucaswflowtreed71c8acbbc76834c3ab0e5c686ed4b664b054ddftargetblankd71c8aca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/cancer-genomics/reproduce_lucas_wflow/tree/d71c8acbbc76834c3ab0e5c686ed4b664b054ddf" target="_blank">d71c8ac</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory

Untracked files:
    Untracked:  CITATION
    Untracked:  temp.rds

Unstaged changes:
    Modified:   .Rprofile
    Modified:   README.Rmd
    Modified:   code/model_code/predict-lucas_c1.r
    Modified:   code/models_c1/model_full_lasso2.rds
    Modified:   code/models_c1/model_seq_glm.rds
    Modified:   data/delfi_sensitivities.csv

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/SessionInfo.Rmd</code>) and HTML
(<code>docs/SessionInfo.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/cancer-genomics/reproduce_lucas_wflow/blob/afb2c3af0e917b7d2079f2701d36a2eb30e68e5d/analysis/SessionInfo.Rmd" target="_blank">afb2c3a</a>
</td>
<td>
shay-279
</td>
<td>
2021-08-19
</td>
<td>
Mathios et. al.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/cancer-genomics/reproduce_lucas_wflow/afb2c3af0e917b7d2079f2701d36a2eb30e68e5d/docs/SessionInfo.html" target="_blank">afb2c3a</a>
</td>
<td>
shay-279
</td>
<td>
2021-08-19
</td>
<td>
Mathios et. al.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>#2
library(ggplot2)
library(magrittr)
library(tidyverse)</code></pre>
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ lubridate 1.9.3     ✔ tibble    3.2.1
✔ purrr     1.0.2     ✔ tidyr     1.3.1
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ tidyr::extract()   masks magrittr::extract()
✖ dplyr::filter()    masks stats::filter()
✖ dplyr::lag()       masks stats::lag()
✖ purrr::set_names() masks magrittr::set_names()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
<pre class="r"><code>library(fs)
library(grid)
library(data.table)</code></pre>
<pre><code>
Attaching package: &#39;data.table&#39;

The following objects are masked from &#39;package:lubridate&#39;:

    hour, isoweek, mday, minute, month, quarter, second, wday, week,
    yday, year

The following objects are masked from &#39;package:dplyr&#39;:

    between, first, last

The following object is masked from &#39;package:purrr&#39;:

    transpose</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;

The following object is masked from &#39;package:lubridate&#39;:

    stamp</code></pre>
<pre class="r"><code>library(devtools)</code></pre>
<pre><code>Loading required package: usethis</code></pre>
<pre class="r"><code>library(here)</code></pre>
<pre><code>here() starts at /Users/rscharpf/Downloads/reproduce_lucas_wflow</code></pre>
<pre class="r"><code>library(tidyverse)
library(openxlsx)
library(devtools)
library(ComplexHeatmap)</code></pre>
<pre><code>========================================
ComplexHeatmap version 2.18.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================</code></pre>
<pre class="r"><code>library(circlize)</code></pre>
<pre><code>========================================
circlize version 0.4.16
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================</code></pre>
<pre class="r"><code>library(data.table)
library(here)
library(tidyverse)
library(data.table)
library(grid)
library(gridExtra)</code></pre>
<pre><code>
Attaching package: &#39;gridExtra&#39;

The following object is masked from &#39;package:dplyr&#39;:

    combine</code></pre>
<pre class="r"><code>library(here)
library(tidyverse)
library(caret)</code></pre>
<pre><code>Loading required package: lattice

Attaching package: &#39;caret&#39;

The following object is masked from &#39;package:purrr&#39;:

    lift</code></pre>
<pre class="r"><code>library(recipes)</code></pre>
<pre><code>
Attaching package: &#39;recipes&#39;

The following object is masked from &#39;package:ComplexHeatmap&#39;:

    prepare

The following object is masked from &#39;package:devtools&#39;:

    check

The following object is masked from &#39;package:stringr&#39;:

    fixed

The following object is masked from &#39;package:stats&#39;:

    step</code></pre>
<pre class="r"><code>library(pROC)</code></pre>
<pre><code>Type &#39;citation(&quot;pROC&quot;)&#39; for a citation.

Attaching package: &#39;pROC&#39;

The following objects are masked from &#39;package:stats&#39;:

    cov, smooth, var</code></pre>
<pre class="r"><code>library(devtools)
library(reshape2)</code></pre>
<pre><code>
Attaching package: &#39;reshape2&#39;

The following objects are masked from &#39;package:data.table&#39;:

    dcast, melt

The following object is masked from &#39;package:tidyr&#39;:

    smiths</code></pre>
<pre class="r"><code>library(plyr)</code></pre>
<pre><code>------------------------------------------------------------------------------
You have loaded plyr after dplyr - this is likely to cause problems.
If you need functions from both plyr and dplyr, please load plyr first, then dplyr:
library(plyr); library(dplyr)
------------------------------------------------------------------------------

Attaching package: &#39;plyr&#39;

The following object is masked from &#39;package:here&#39;:

    here

The following objects are masked from &#39;package:dplyr&#39;:

    arrange, count, desc, failwith, id, mutate, rename, summarise,
    summarize

The following object is masked from &#39;package:purrr&#39;:

    compact</code></pre>
<pre class="r"><code>library(gridExtra)
library(grid)
library(here)

#3
library(SummarizedExperiment)</code></pre>
<pre><code>Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#39;matrixStats&#39;

The following object is masked from &#39;package:plyr&#39;:

    count

The following object is masked from &#39;package:dplyr&#39;:

    count


Attaching package: &#39;MatrixGenerics&#39;

The following objects are masked from &#39;package:matrixStats&#39;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#39;BiocGenerics&#39;

The following object is masked from &#39;package:pROC&#39;:

    var

The following object is masked from &#39;package:gridExtra&#39;:

    combine

The following object is masked from &#39;package:fs&#39;:

    path

The following objects are masked from &#39;package:lubridate&#39;:

    intersect, setdiff, union

The following objects are masked from &#39;package:dplyr&#39;:

    combine, intersect, setdiff, union

The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, aperm, append, as.data.frame, basename, cbind,
    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,
    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,
    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,
    Position, rank, rbind, Reduce, rownames, sapply, setdiff, sort,
    table, tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#39;S4Vectors&#39;

The following object is masked from &#39;package:plyr&#39;:

    rename

The following objects are masked from &#39;package:data.table&#39;:

    first, second

The following objects are masked from &#39;package:lubridate&#39;:

    second, second&lt;-

The following objects are masked from &#39;package:dplyr&#39;:

    first, rename

The following object is masked from &#39;package:tidyr&#39;:

    expand

The following object is masked from &#39;package:utils&#39;:

    findMatches

The following objects are masked from &#39;package:base&#39;:

    expand.grid, I, unname

Loading required package: IRanges

Attaching package: &#39;IRanges&#39;

The following object is masked from &#39;package:plyr&#39;:

    desc

The following object is masked from &#39;package:data.table&#39;:

    shift

The following object is masked from &#39;package:lubridate&#39;:

    %within%

The following objects are masked from &#39;package:dplyr&#39;:

    collapse, desc, slice

The following object is masked from &#39;package:purrr&#39;:

    reduce

Loading required package: GenomeInfoDb

Attaching package: &#39;GenomicRanges&#39;

The following object is masked from &#39;package:magrittr&#39;:

    subtract

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.


Attaching package: &#39;Biobase&#39;

The following object is masked from &#39;package:MatrixGenerics&#39;:

    rowMedians

The following objects are masked from &#39;package:matrixStats&#39;:

    anyMissing, rowMedians</code></pre>
<pre class="r"><code>library(devtools)
library(magrittr)
library(stringr)
library(tidyverse)
library(grid)
library(lemon)</code></pre>
<pre><code>
Attaching package: &#39;lemon&#39;

The following object is masked from &#39;package:purrr&#39;:

    %||%</code></pre>
<pre class="r"><code>library(gridExtra)
library(cowplot)
library(data.table)
library(pROC)
library(RColorBrewer)
library(here)


#4
library(readxl)
library(ggplot2)
library(here)
library(SummarizedExperiment)
library(tidyverse)
library(magrittr)
library(stringr)
library(survival)</code></pre>
<pre><code>
Attaching package: &#39;survival&#39;

The following object is masked from &#39;package:caret&#39;:

    cluster</code></pre>
<pre class="r"><code>library(survminer)</code></pre>
<pre><code>Loading required package: ggpubr

Attaching package: &#39;ggpubr&#39;

The following object is masked from &#39;package:plyr&#39;:

    mutate

The following object is masked from &#39;package:cowplot&#39;:

    get_legend


Attaching package: &#39;survminer&#39;

The following object is masked from &#39;package:survival&#39;:

    myeloma</code></pre>
<pre class="r"><code>library(ggplot2)
library(readxl)
library(here)
library(devtools)

#5
library(ggplot2)
library(here)
library(ggplot2)
library(pheatmap)</code></pre>
<pre><code>
Attaching package: &#39;pheatmap&#39;

The following object is masked from &#39;package:ComplexHeatmap&#39;:

    pheatmap</code></pre>
<pre class="r"><code>library(here)
library(dplyr)
library(tidyverse)
library(ggplot2)
library(pROC)
library(purrr)
library(here)

#6
library(RColorBrewer)
library(ggplot2)
library(tidyverse)
library(dplyr)
library(readxl)
library(magrittr)
library(grid)
library(gridExtra)
library(here)
library(hrbrthemes)</code></pre>
<pre><code>NOTE: Either Arial Narrow or Roboto Condensed fonts are required to use these themes.
      Please use hrbrthemes::import_roboto_condensed() to install Roboto Condensed and
      if Arial Narrow is not on your system, please see https://bit.ly/arialnarrow</code></pre>
<pre class="r"><code>library(viridis)</code></pre>
<pre><code>Loading required package: viridisLite</code></pre>
<pre class="r"><code>library(kableExtra)</code></pre>
<pre><code>
Attaching package: &#39;kableExtra&#39;

The following object is masked from &#39;package:dplyr&#39;:

    group_rows</code></pre>
<pre class="r"><code>library(jcolors)
library(ggplot2)
library(magrittr)
library(tidyverse)
library(fs)
library(grid)
library(mvtnorm)
library(readxl)
library(kableExtra)
library(here)

#extended 2
library(tidyverse)
library(ggplot2)
library(SummarizedExperiment)
library(paletteer)
library(ggsci)
library(here)
library(devtools)
library(RColorBrewer)
library(SummarizedExperiment)
library(magrittr)
library(tidyverse)
library(ggplot2)
library(here)

#extended 3
library(tidyverse)
library(caret)
library(recipes)
library(pROC)
library(devtools)
library(glmnet)</code></pre>
<pre><code>Loading required package: Matrix

Attaching package: &#39;Matrix&#39;

The following object is masked from &#39;package:S4Vectors&#39;:

    expand

The following objects are masked from &#39;package:tidyr&#39;:

    expand, pack, unpack

Loaded glmnet 4.1-8</code></pre>
<pre class="r"><code>library(reshape2)
library(here)

#extended 4
library(tidyverse)
library(magrittr)
library(stringr)
library(gridExtra)
library(broom)
library(SummarizedExperiment)
library(devtools)
library(here)

#extended 5
library(tidyverse)
library(magrittr)
library(stringr)
library(devtools)
library(here)

#extended 6
library(pROC)
library(readxl)
library(here)
library(readxl)
library(tidyverse)
library(caret)
library(pROC)
library(precrec)</code></pre>
<pre><code>
Attaching package: &#39;precrec&#39;

The following object is masked from &#39;package:pROC&#39;:

    auc</code></pre>
<pre class="r"><code>library(here)
library(RColorBrewer)
library(SummarizedExperiment)
library(ggplot2)
library(tidyverse)
library(dplyr)
library(magrittr)
library(here)
library(devtools)


#extended 7
library(devtools)
library(data.table)
library(caret)
library(recipes)
library(magrittr)
library(tidyr)
library(purrr)
library(cowplot)
library(devtools)
library(here)

#extended 8
library(tidyverse)
library(magrittr)
library(lemon)
library(broom)
library(stringr)
library(SummarizedExperiment)
library(devtools)
library(here)

#extended 9
library(SummarizedExperiment)
library(devtools)
library(magrittr)
library(stringr)
library(tidyverse)
library(grid)
library(lemon)
library(gridExtra)
library(cowplot)
library(data.table)
library(pROC)
library(RColorBrewer)
library(here)

#extended 10
library(here)
library(ggplot2)
library(dplyr)
library(ggplot2)
library(pROC)
library(here)
library(dplyr)
library(ggplot2)
library(pROC)
library(here)

#extended 11
library(SummarizedExperiment)
library(tidyverse)
library(magrittr)
library(stringr)
library(survival)
library(survminer)
library(ggplot2)
library(readxl)
library(tidyr)
library(broom)
library(gridExtra)
library(kableExtra)
library(tidyr)
library(devtools)
library(here)

#extended 12
library(SummarizedExperiment)
library(tidyverse)
library(magrittr)
library(stringr)
library(survival)
library(survminer)
library(ggplot2)
library(readxl)
library(tidyr)
library(broom)
library(gridExtra)
library(kableExtra)
library(tidyr)
library(devtools)
library(here)

#models
library(tidyverse)
library(caret)
library(recipes)
library(pROC)
library(devtools)
library(here)
library(tidyverse)
library(caret)
library(recipes)
library(pROC)
library(devtools)
library(openxlsx)
library(precrec)
library(tidyverse)
library(caret)
library(pROC)
library(readxl)
library(DT)
library(recipes)
library(devtools)
library(data.table)
library(here)
library(devtools)
library(here)
library(tidyverse)
library(caret)

#Delfi Pipeline
suppressMessages(library(GenomicAlignments))
suppressMessages(library(GenomicRanges))
suppressMessages(library(Rsamtools))
suppressMessages(library(devtools))
suppressMessages(library(Biostrings))
suppressMessages(library(data.table))
suppressMessages(library(getopt))
suppressMessages(library(devtools))
library(PlasmaTools.lucas)</code></pre>
<pre><code>Loading required package: biovizBase</code></pre>
<pre class="r"><code>library(PlasmaToolsHiseq.hg19)</code></pre>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.2 (2023-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.3.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] PlasmaToolsHiseq.hg19_0.0.0.9000 PlasmaTools.lucas_0.0.0.9100    
 [3] biovizBase_1.50.0                getopt_1.20.4                   
 [5] GenomicAlignments_1.38.2         Rsamtools_2.18.0                
 [7] Biostrings_2.70.2                XVector_0.42.0                  
 [9] DT_0.32                          precrec_0.14.4                  
[11] broom_1.0.5                      glmnet_4.1-8                    
[13] Matrix_1.6-5                     ggsci_3.0.0                     
[15] paletteer_1.6.0                  mvtnorm_1.2-4                   
[17] jcolors_0.0.5                    kableExtra_1.4.0                
[19] viridis_0.6.5                    viridisLite_0.4.2               
[21] hrbrthemes_0.8.0                 pheatmap_1.0.12                 
[23] survminer_0.4.9                  ggpubr_0.6.0                    
[25] survival_3.5-8                   readxl_1.4.3                    
[27] RColorBrewer_1.1-3               lemon_0.4.9                     
[29] SummarizedExperiment_1.32.0      Biobase_2.62.0                  
[31] GenomicRanges_1.54.1             GenomeInfoDb_1.38.6             
[33] IRanges_2.36.0                   S4Vectors_0.40.2                
[35] BiocGenerics_0.48.1              MatrixGenerics_1.14.0           
[37] matrixStats_1.2.0                plyr_1.8.9                      
[39] reshape2_1.4.4                   pROC_1.18.5                     
[41] recipes_1.0.10                   caret_6.0-94                    
[43] lattice_0.22-5                   gridExtra_2.3                   
[45] circlize_0.4.16                  ComplexHeatmap_2.18.0           
[47] openxlsx_4.2.5.2                 here_1.0.1                      
[49] devtools_2.4.5                   usethis_2.2.3                   
[51] cowplot_1.1.3                    data.table_1.15.0               
[53] fs_1.6.3                         lubridate_1.9.3                 
[55] forcats_1.0.0                    stringr_1.5.1                   
[57] dplyr_1.1.4                      purrr_1.0.2                     
[59] readr_2.1.5                      tidyr_1.3.1                     
[61] tibble_3.2.1                     tidyverse_2.0.0                 
[63] magrittr_2.0.3                   ggplot2_3.5.0                   
[65] workflowr_1.7.1                 

loaded via a namespace (and not attached):
  [1] ProtGenerics_1.34.0      bitops_1.0-7             httr_1.4.7              
  [4] doParallel_1.0.17        profvis_0.3.8            tools_4.3.2             
  [7] backports_1.4.1          utf8_1.2.4               R6_2.5.1                
 [10] lazyeval_0.2.2           GetoptLong_1.0.5         urlchecker_1.0.1        
 [13] withr_3.0.0              prettyunits_1.2.0        cli_3.6.2               
 [16] sass_0.4.8               survMisc_0.5.6           systemfonts_1.0.5       
 [19] foreign_0.8-86           gfonts_0.2.0             svglite_2.1.3           
 [22] dichromat_2.0-0.1        parallelly_1.37.0        sessioninfo_1.2.2       
 [25] BSgenome_1.70.2          RSQLite_2.3.5            rstudioapi_0.15.0       
 [28] httpcode_0.3.0           BiocIO_1.12.0            generics_0.1.3          
 [31] shape_1.4.6.1            car_3.1-2                zip_2.3.1               
 [34] fansi_1.0.6              abind_1.4-5              lifecycle_1.0.4         
 [37] whisker_0.4.1            yaml_2.3.8               carData_3.0-5           
 [40] BiocFileCache_2.10.1     SparseArray_1.2.4        blob_1.2.4              
 [43] promises_1.2.1           crayon_1.5.2             miniUI_0.1.1.1          
 [46] GenomicFeatures_1.54.3   KEGGREST_1.42.0          pillar_1.9.0            
 [49] knitr_1.45               rjson_0.2.21             future.apply_1.11.1     
 [52] codetools_0.2-19         glue_1.7.0               getPass_0.2-4           
 [55] fontLiberation_0.1.0     remotes_2.4.2.1          vctrs_0.6.5             
 [58] png_0.1-8                cellranger_1.1.0         gtable_0.3.4            
 [61] rematch2_2.1.2           cachem_1.0.8             gower_1.0.1             
 [64] xfun_0.42                S4Arrays_1.2.0           mime_0.12               
 [67] prodlim_2023.08.28       timeDate_4032.109        iterators_1.0.14        
 [70] KMsurv_0.1-5             hardhat_1.3.1            lava_1.7.3              
 [73] ellipsis_0.3.2           ipred_0.9-14             nlme_3.1-164            
 [76] bit64_4.0.5              fontquiver_0.2.1         filelock_1.0.3          
 [79] progress_1.2.3           rprojroot_2.0.4          bslib_0.6.1             
 [82] rpart_4.1.23             DBI_1.2.2                Hmisc_5.1-1             
 [85] colorspace_2.1-0         nnet_7.3-19              tidyselect_1.2.0        
 [88] processx_3.8.3           bit_4.0.5                compiler_4.3.2          
 [91] extrafontdb_1.0          curl_5.2.0               git2r_0.33.0            
 [94] htmlTable_2.4.2          xml2_1.3.6               fontBitstreamVera_0.1.1 
 [97] DelayedArray_0.28.0      rtracklayer_1.62.0       checkmate_2.3.1         
[100] scales_1.3.0             callr_3.7.5              rappdirs_0.3.3          
[103] digest_0.6.34            rmarkdown_2.25           base64enc_0.1-3         
[106] htmltools_0.5.7          pkgconfig_2.0.3          extrafont_0.19          
[109] ensembldb_2.26.0         dbplyr_2.4.0             highr_0.10              
[112] fastmap_1.1.1            rlang_1.1.3              GlobalOptions_0.1.2     
[115] htmlwidgets_1.6.4        shiny_1.8.0              jquerylib_0.1.4         
[118] zoo_1.8-12               jsonlite_1.8.8           BiocParallel_1.36.0     
[121] ModelMetrics_1.2.2.2     VariantAnnotation_1.48.1 RCurl_1.98-1.14         
[124] Formula_1.2-5            GenomeInfoDbData_1.2.11  munsell_0.5.0           
[127] Rcpp_1.0.12              gdtools_0.3.6            stringi_1.8.3           
[130] zlibbioc_1.48.0          MASS_7.3-60.0.1          pkgbuild_1.4.3          
[133] parallel_4.3.2           listenv_0.9.1            splines_4.3.2           
[136] hms_1.1.3                ps_1.7.6                 ggsignif_0.6.4          
[139] biomaRt_2.58.2           pkgload_1.3.4            XML_3.99-0.16.1         
[142] crul_1.4.0               evaluate_0.23            tzdb_0.4.0              
[145] foreach_1.5.2            httpuv_1.6.14            Rttf2pt1_1.3.12         
[148] future_1.33.1            km.ci_0.5-6              clue_0.3-65             
[151] xtable_1.8-4             restfulr_0.0.15          AnnotationFilter_1.26.0 
[154] rstatix_0.7.2            later_1.3.2              class_7.3-22            
[157] AnnotationDbi_1.64.1     memoise_2.0.1            cluster_2.1.6           
[160] timechange_0.3.0         globals_0.16.2          </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.2 (2023-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.3.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] PlasmaToolsHiseq.hg19_0.0.0.9000 PlasmaTools.lucas_0.0.0.9100    
 [3] biovizBase_1.50.0                getopt_1.20.4                   
 [5] GenomicAlignments_1.38.2         Rsamtools_2.18.0                
 [7] Biostrings_2.70.2                XVector_0.42.0                  
 [9] DT_0.32                          precrec_0.14.4                  
[11] broom_1.0.5                      glmnet_4.1-8                    
[13] Matrix_1.6-5                     ggsci_3.0.0                     
[15] paletteer_1.6.0                  mvtnorm_1.2-4                   
[17] jcolors_0.0.5                    kableExtra_1.4.0                
[19] viridis_0.6.5                    viridisLite_0.4.2               
[21] hrbrthemes_0.8.0                 pheatmap_1.0.12                 
[23] survminer_0.4.9                  ggpubr_0.6.0                    
[25] survival_3.5-8                   readxl_1.4.3                    
[27] RColorBrewer_1.1-3               lemon_0.4.9                     
[29] SummarizedExperiment_1.32.0      Biobase_2.62.0                  
[31] GenomicRanges_1.54.1             GenomeInfoDb_1.38.6             
[33] IRanges_2.36.0                   S4Vectors_0.40.2                
[35] BiocGenerics_0.48.1              MatrixGenerics_1.14.0           
[37] matrixStats_1.2.0                plyr_1.8.9                      
[39] reshape2_1.4.4                   pROC_1.18.5                     
[41] recipes_1.0.10                   caret_6.0-94                    
[43] lattice_0.22-5                   gridExtra_2.3                   
[45] circlize_0.4.16                  ComplexHeatmap_2.18.0           
[47] openxlsx_4.2.5.2                 here_1.0.1                      
[49] devtools_2.4.5                   usethis_2.2.3                   
[51] cowplot_1.1.3                    data.table_1.15.0               
[53] fs_1.6.3                         lubridate_1.9.3                 
[55] forcats_1.0.0                    stringr_1.5.1                   
[57] dplyr_1.1.4                      purrr_1.0.2                     
[59] readr_2.1.5                      tidyr_1.3.1                     
[61] tibble_3.2.1                     tidyverse_2.0.0                 
[63] magrittr_2.0.3                   ggplot2_3.5.0                   
[65] workflowr_1.7.1                 

loaded via a namespace (and not attached):
  [1] ProtGenerics_1.34.0      bitops_1.0-7             httr_1.4.7              
  [4] doParallel_1.0.17        profvis_0.3.8            tools_4.3.2             
  [7] backports_1.4.1          utf8_1.2.4               R6_2.5.1                
 [10] lazyeval_0.2.2           GetoptLong_1.0.5         urlchecker_1.0.1        
 [13] withr_3.0.0              prettyunits_1.2.0        cli_3.6.2               
 [16] sass_0.4.8               survMisc_0.5.6           systemfonts_1.0.5       
 [19] foreign_0.8-86           gfonts_0.2.0             svglite_2.1.3           
 [22] dichromat_2.0-0.1        parallelly_1.37.0        sessioninfo_1.2.2       
 [25] BSgenome_1.70.2          RSQLite_2.3.5            rstudioapi_0.15.0       
 [28] httpcode_0.3.0           BiocIO_1.12.0            generics_0.1.3          
 [31] shape_1.4.6.1            car_3.1-2                zip_2.3.1               
 [34] fansi_1.0.6              abind_1.4-5              lifecycle_1.0.4         
 [37] whisker_0.4.1            yaml_2.3.8               carData_3.0-5           
 [40] BiocFileCache_2.10.1     SparseArray_1.2.4        blob_1.2.4              
 [43] promises_1.2.1           crayon_1.5.2             miniUI_0.1.1.1          
 [46] GenomicFeatures_1.54.3   KEGGREST_1.42.0          pillar_1.9.0            
 [49] knitr_1.45               rjson_0.2.21             future.apply_1.11.1     
 [52] codetools_0.2-19         glue_1.7.0               getPass_0.2-4           
 [55] fontLiberation_0.1.0     remotes_2.4.2.1          vctrs_0.6.5             
 [58] png_0.1-8                cellranger_1.1.0         gtable_0.3.4            
 [61] rematch2_2.1.2           cachem_1.0.8             gower_1.0.1             
 [64] xfun_0.42                S4Arrays_1.2.0           mime_0.12               
 [67] prodlim_2023.08.28       timeDate_4032.109        iterators_1.0.14        
 [70] KMsurv_0.1-5             hardhat_1.3.1            lava_1.7.3              
 [73] ellipsis_0.3.2           ipred_0.9-14             nlme_3.1-164            
 [76] bit64_4.0.5              fontquiver_0.2.1         filelock_1.0.3          
 [79] progress_1.2.3           rprojroot_2.0.4          bslib_0.6.1             
 [82] rpart_4.1.23             DBI_1.2.2                Hmisc_5.1-1             
 [85] colorspace_2.1-0         nnet_7.3-19              tidyselect_1.2.0        
 [88] processx_3.8.3           bit_4.0.5                compiler_4.3.2          
 [91] extrafontdb_1.0          curl_5.2.0               git2r_0.33.0            
 [94] htmlTable_2.4.2          xml2_1.3.6               fontBitstreamVera_0.1.1 
 [97] DelayedArray_0.28.0      rtracklayer_1.62.0       checkmate_2.3.1         
[100] scales_1.3.0             callr_3.7.5              rappdirs_0.3.3          
[103] digest_0.6.34            rmarkdown_2.25           base64enc_0.1-3         
[106] htmltools_0.5.7          pkgconfig_2.0.3          extrafont_0.19          
[109] ensembldb_2.26.0         dbplyr_2.4.0             highr_0.10              
[112] fastmap_1.1.1            rlang_1.1.3              GlobalOptions_0.1.2     
[115] htmlwidgets_1.6.4        shiny_1.8.0              jquerylib_0.1.4         
[118] zoo_1.8-12               jsonlite_1.8.8           BiocParallel_1.36.0     
[121] ModelMetrics_1.2.2.2     VariantAnnotation_1.48.1 RCurl_1.98-1.14         
[124] Formula_1.2-5            GenomeInfoDbData_1.2.11  munsell_0.5.0           
[127] Rcpp_1.0.12              gdtools_0.3.6            stringi_1.8.3           
[130] zlibbioc_1.48.0          MASS_7.3-60.0.1          pkgbuild_1.4.3          
[133] parallel_4.3.2           listenv_0.9.1            splines_4.3.2           
[136] hms_1.1.3                ps_1.7.6                 ggsignif_0.6.4          
[139] biomaRt_2.58.2           pkgload_1.3.4            XML_3.99-0.16.1         
[142] crul_1.4.0               evaluate_0.23            tzdb_0.4.0              
[145] foreach_1.5.2            httpuv_1.6.14            Rttf2pt1_1.3.12         
[148] future_1.33.1            km.ci_0.5-6              clue_0.3-65             
[151] xtable_1.8-4             restfulr_0.0.15          AnnotationFilter_1.26.0 
[154] rstatix_0.7.2            later_1.3.2              class_7.3-22            
[157] AnnotationDbi_1.64.1     memoise_2.0.1            cluster_2.1.6           
[160] timechange_0.3.0         globals_0.16.2          </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
