---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

## Detection and characterization of lung cancer using cell-free DNA fragmentomes

Abstract:

Lung cancer remains the leading cause of cancer death world-wide, largely due to its late diagnosis.  Non-invasive approaches for assessment of cell-free DNA (cfDNA) provide an opportunity for early detection and intervention that may have broader accessibility than current imaging approaches.  Using a machine learning model for detecting tumor-derived cfDNA through genome-wide analyses of cfDNA fragmentation, we examined a prospective study of 365 individuals at risk for lung cancer (Lung Cancer Diagnostic Study, LUCAS), including 129 individuals ultimately diagnosed with lung cancer and 236 individuals determined to not have lung cancer.  We externally validated the cancer detection model using an independent cohort of 385 non-cancer individuals and 46 predominantly early stage lung cancer patients.  Combining fragmentation features with clinical risk factors and CEA levels followed by CT imaging detected 94% of patients with cancer across stages and subtypes, including 91% of stage I/II and 96% of stage III/IV, at 80% specificity.  Genome-wide fragmentation profiles across ~13,000 ASCL1 transcription factor binding sites could be used to distinguish individuals with small cell lung cancer (SCLC) from those with non-small cell lung cancer (NSCLC) with high accuracy (AUC=0.98).  Among individuals with lung cancer, a higher cfDNA fragmentation score was associated with tumor size and invasion, and represented an independent prognostic indicator of survival.  These studies provide a facile approach for non-invasive detection of lung cancer and clinical management of this disease.
</center>


# Dependencies

A list of R-related dependencies for reproducing the figures in the LUCAS manuscript is available [here](SessionInfo.html).  Most of the R packages are available from Bioconductor or CRAN.  PlasmaTools.lucas and rlucas are available in the code/ folder.


# Alignment and pre-processing

## Alignment

The alignment of the sequenced fragments and pre-processing of the aligned reads are outside the scope of this workflowr repository.  For processing unaligned reads, we used  fastp (v0.20.0), bowtie2 (v 2.3.5.1),  samtools (v 1.9), and sambamba (v 0.7.1).  The following scripts that can be adapted for re-analysis of the LUCAS dataset:

- [align.sh](../code/preprocessing/align.sh):  aligns sequenced fragments to the Hg19 reference genome using bowtie2. Duplicates are flagged and removed with sambamba.

## Preprocessing

See the [code/preprocessing/README](../code/preprocessing/README.Rmd) for details.

# Figures

- Figure 1: [[png](figure/fig1.png) ]

- Figure 2:

  - 2a [[R](../analysis/fig2a.Rmd), [html](fig2a.html) , [pdf](figure/fig2a.Rmd/fig2a-1.pdf) ]

  - 2b [[R](../analysis/fig2b.Rmd),[html](fig2b.html) , [pdf](figure/fig2b.Rmd/heatmap-1.pdf) ]

  - 2c (in two panels) [[R](../analysis/fig2c.Rmd),[html](fig2c.html) , [pdf](figure/fig2c.Rmd/heatmap3-1.pdf), [R](../analysis/fig2c_pt2.Rmd), [html](fig2c_pt2.html) , [pdf](figure/fig2c_pt2.Rmd/plot-1.pdf)  ]

- Figure 3:[[R](../analysis/fig3.Rmd), [html](fig3.html)]

  - 3a [[pdf](figure/fig3.Rmd/fig3A-1.pdf) ]

  - 3b (in two parts) [[pdf](figure/fig3.Rmd/roc_categories-1.pdf), [pdf](figure/fig3.Rmd/overall-1.pdf)]

  - 3c [[pdf](figure/fig3.Rmd/fig3C-1.pdf) ]

- Figure 4:

  - 4a, 4b (in two parts) [[R](../analysis/fig4ab.Rmd),[html](fig4ab.html),  [pdf](figure/fig4ab.Rmd/fig4a-1.pdf), [pdf](figure/fig4ab.Rmd/fig4b-1.pdf)]

  - 4c [[R](../analysis/fig4c.Rmd), [html](fig4c.html) , [pdf](figure/fig4c.Rmd/survcurve_seq-1.pdf)]

- Figure 5:

  - 5a [[R](../analysis/fig5a.Rmd), [html](fig5a.html) , [pdf](figure/fig5a.Rmd/ascl1_expression-1.pdf)]

  - 5b [[R](../analysis/fig5b.Rmd), [html](fig5b.html) , [pdf](figure/fig5b.Rmd/fig2-1.pdf)]

  - 5c, 5d, 5e, 5f (in one panel) [[R](../analysis/fig5cdef.Rmd), [html](fig5cdef.html) , [pdf](figure/fig5cdef.Rmd/daniel_composite-1.pdf)]

- Figure 6:

  - 6a [[png](figure/fig6a.png) ]

  - 6b [[R](../analysis/fig6b.Rmd), [html](fig6b.html) , [pdf](figure/fig6b.Rmd/sens_table-1.pdf)]

  - 6c, 6d, 6e, 6f, 6g (in one panel) [[R](../analysis/fig6cdefg.Rmd), [html](fig6cdefg.html) , [pdf](figure/fig6cdefg.Rmd/composite-1.pdf)]

Extended Figures:

- Extended Figure 1: [[png](figure/extended_fig1.png)]


- Extended Figure 2:

  - 2a, 2b (in one panel) [[R](../analysis/extended_fig2ab.Rmd), [html](extended_fig2ab.html) , [pdf](figure/extended_fig2ab.Rmd/pcafig-1.pdf)]

  - 2c [[R](../analysis/extended_fig2c.Rmd), [html](extended_fig2c.html) , [pdf](figure/extended_fig2c.Rmd/profiles-1.pdf)]

- Extended Figure 3: [[R](../analysis/extended_fig3.Rmd), [html](extended_fig3.html) , [pdf](figure/extended_fig3.Rmd/heatmap-1.pdf)]

- Extended Figure 4: [[R](../analysis/extended_fig4.Rmd), [html](extended_fig4.html)]

  - 4a [[pdf](figure/extended_fig4.Rmd/smoking_seqmodel-1.pdf)]

  - 4b, 4c (in one panel) [[pdf](figure/extended_fig4.Rmd/gridgraphic-1.pdf)]

  - 4d [[pdf](figure/extended_fig4.Rmd/age_seqmodel-1.pdf)]

- Extended Figure 5: [[R](../analysis/extended_fig5.Rmd), [html](extended_fig5.html) , [pdf](figure/extended_fig5.Rmd/protein_scatter-1.pdf)]

- Extended Figure 6:

  - 6a [[R](../analysis/extended_fig6a.Rmd), [html](extended_fig6a.html) , [pdf](figure/extended_fig6a.Rmd/unnamed-chunk-1-1.pdf)]

  - 6b [[R](../analysis/extended_fig6b.Rmd), [html](extended_fig6b.html) , [pdf](figure/extended_fig6b.Rmd/packages-4.pdf)]

  - 6c [[R](../analysis/extended_fig6c.Rmd), [html](extended_fig6c.html) , [pdf](figure/extended_fig6c.Rmd/medians-1.pdf)]

- Extended Figure 7: [[R](../analysis/extended_fig7.Rmd), [html](extended_fig7.html) , [pdf](figure/extended_fig7.Rmd/extended_fig5-1.pdf)]

- Extended Figure 8: [[R](../analysis/extended_fig8.Rmd), [html](extended_fig8.html) , [pdf](figure/extended_fig8.Rmd/cea_nlratio-1.pdf)]

- Extended Figure 9 [[R](../analysis/extended_fig9.Rmd), [html](extended_fig9.html) , [pdf](figure/extended_fig9.Rmd/extfig9-1.pdf)]

- Extended Figure 10:

  - 10a [[R](../analysis/extended_fig10a.Rmd), [html](extended_fig10a.html),[pdf](figure/extended_fig10a.Rmd/tfbs_fragment_size_10_sclc_vs_10_no_cancer-1.pdf)]

  - 10b [[R](../analysis/extended_fig10b.Rmd), [html](extended_fig10ab.html),[pdf](figure/extended_fig10ab.Rmd/sclc_no_cancer_no_benign-1.pdf)]

  - 10c, 10d (in two parts) [[R](../analysis/extended_fig10cd.Rmd), [html](extended_fig10cd.html), [pdf](figure/extended_fig10cd.Rmd/tfbs_fragment_size_sclc_vs_other-1.pdf), [pdf](figure/extended_fig10cd.Rmd/roc_fragment_size_sclc_vs_other-1.pdf)]

- Extended Figure 11 [[R](../analysis/extended_fig11.Rmd), [html](extended_fig11.html) , [pdf](figure/extended_fig11.Rmd/composite-1.pdf)]

- Extended Figure 12 [[R](../analysis/extended_fig12.Rmd), [html](extended_fig12.html)]

  - 12a [[pdf](figure/extended_fig12.Rmd/fig12a-1.pdf)]

  - 12b [[pdf](figure/extended_fig12.Rmd/survcurve_seq-1.pdf)]


[[SessionInfo](SessionInfo.html)] (packages and versions needed to implement the analysis)
